card/mod/05_standard/spec/set/right/account_spec.rb

```diff
@@ -145,11 +145,11 @@
     context 'denied access' do
       it 'excludes protected subcards' do
         skip
-        Card.create(:name=>"A+B+*self+*read", :type=>'Pointer', :content=>"[[u1]]")
-        u2 = Card.fetch 'u2+*following', :new=>{:type=>'Pointer'}
+        Card.create(name: "A+B+*self+*read", type: 'Pointer', content: "[[u1]]")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.